intake_stac/catalog.py

```diff
@@ -13,6 +13,7 @@
 
 class AbstractStacCatalog(Catalog):
 
+    container = 'python'
```

Comment:
The correct container for these is "catalog", which they would previously already have had since `Catalog.container` is "catalog". Were you meaning to imply that the outputs would be "python"-type? Then, it would go on the bottom-most generation of items - but I thought they would be xarrays.

Reply:
ah! I get it. I was misunderstanding the meaning of container. Probably just poor reading comprehension on my part. Thanks.